test/e2e/operator_test.go

```diff
@@ -81,6 +81,7 @@ var (
 		{Type: operatorv1.LoadBalancerReadyIngressConditionType, Status: operatorv1.ConditionTrue},
 		{Type: operatorv1.DNSManagedIngressConditionType, Status: operatorv1.ConditionTrue},
 		{Type: operatorv1.DNSReadyIngressConditionType, Status: operatorv1.ConditionTrue},
+		{Type: operatorv1.OperatorStatusTypeProgressing, Status: operatorv1.ConditionFalse},
```

Comment:
I think one of the tests failed once or twice (when updating from `Umanaged` to `Managed`) because the condition was satisfied even though the operator was still progressing.